### PR TITLE
feat(hub): adding iam invite banner

### PR DESCRIPTION
**Pull request by BrunoMarquesOVH**
ref: MANAGER-12081

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12081
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits

## Description

Adding IAM invite banner

## Related

- [x] CMT-3057

### CODE REVIEW

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 1]
```diff
@@ -59,6 +59,28 @@
                 ></span>
             </a>
         </oui-message>
+        <!-- IAM Banner-->
+        <oui-message
+            data-type="info"
+            data-on-dismiss="$ctrl.onDismiss()"
+            data-dismissable
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/dashboard.html:66]: **nitpick:** `onDismiss` is not defined in `packages/manager/apps/hub/src/dashboard/dashboard.controller.js`

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.constant.js [hunk 2]
```diff
@@ -10,11 +10,17 @@ export const KYC_TRACKING_PREFIX = 'alert::kyc-india';
 
 export const KYC_HIT_PREFIX = `${KYC_TRACKING_PREFIX}::verify-identity`;
 
+export const IAM_TRACKING_PREFIX = 'alert::iam-availability';
+
+export const IAM_HIT_PREFIX = 'alert::iam-availability::goto-iam';
```
Comment (ghyenne) [inline packages/manager/apps/hub/src/dashboard/dashboard.constant.js:15]:
```suggestion
export const IAM_HIT_PREFIX = `${IAM_TRACKING_PREFIX}::goto-iam';
```

File: packages/manager/apps/hub/src/dashboard/dashboard.html (hunk 1 above)
Comment (BrunoMarquesOVH) [reply packages/manager/apps/hub/src/dashboard/dashboard.html]: Ok, it was like this on other banners but I fixed it and removed it. Thx

File: packages/manager/apps/hub/src/dashboard/dashboard.constant.js (hunk 2 above)
Comment (BrunoMarquesOVH) [reply packages/manager/apps/hub/src/dashboard/dashboard.constant.js]: Ok done!

**Review by JacquesLarique — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by darsene — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by qpavy — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**